tox.ini

```diff
@@ -55,7 +55,7 @@ commands =
     examples: {[testenv:base-command]commands} examples/test/ -m examples
     all: {[testenv:base-command]commands} test/ examples/test/
     manual: {[testenv:base-command]commands}
-
+    
```

Comment:
Did your IDE do this?
I'd prefer not to have a line of extra whitespace like this.